core/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
 - Hide `prost::Error` from public API in `FromEnvelopeError::InvalidPeerRecord` and `signed_envelope::DecodingError`. See [PR 3058].
 
+- Fixed minor clippy issue. See [PR 3106].
```

Comment:
I don't think we need to announce clippy improvements in the `CHANGELOG.md` if they don't affect the public API, see [here](https://github.com/libp2p/rust-libp2p/pull/3081/files)

Reply:
Agreed, lets not clutter the changelog with such details.

Reply:
I removed the changes from the changelog.